getRecord(id)
.then(data => {
    console.log(data)
})
.catch(error => {
    console.error(error);
});